riva/clients/asr/riva_asr_client.cc

```diff
@@ -456,6 +459,7 @@ main(int argc, char** argv)
   str_usage << "           --boosted_words_score=<float>" << std::endl;
   str_usage << "           --ssl_cert=<filename>" << std::endl;
   str_usage << "           --speaker_diarization=<true|false>" << std::endl;
+  str_usage << "           --max_speaker_count=<int>" << std::endl;
```

Comment:
should we name this field as `diarization_max_speakers` to be clear?

Reply:
Yeah, makes sense. Fixed